lib/ansible/modules/include_role.py

```diff
@@ -83,6 +89,8 @@
     diff_mode:
         support: none
 notes:
+  - Begining in ansible 2.21 we have normalized how errors from the include itself are emitted.
```

Comment:
I think it's still inconsistent because we don't give an error for invalid _from fields if the directory (tasks/vars/etc) does not exist at all in the role.

Reply:
not sure i follow can you show an example?

Reply:
```yaml
- hosts: localhost
  gather_facts: no
  tasks:
  - file:
      path: roles/rolename
      state: "{{ item }}"
    loop:
      - absent
      - directory

  - include_role:  # does not fail because the role doesn't contain a tasks directory
      name: rolename
      tasks_from: missing.yml
```

Reply:
that is because 'missing.yml' CAN be in role/ dir, as role search falls back to that, it should error out when missing.yml is not found. 

Just ran, I see that it just doesn't fail at all, yeah, that is a bug, but that is also previous to this PR, i'll fix it here also

Reply:
I just thought the phrasing here could be misunderstood since there are multiple inconsistencies.

```suggestion
  - Beginning in ansible 2.21 we have normalized how error types from the include itself are emitted.
```